packages/mobile/src/analytics/Events.tsx

```diff
@@ -364,6 +364,7 @@ export enum FiatExchangeEvents {
   // Add fund flow entered through home screen cash in bottom sheet
   cico_add_funds_bottom_sheet_selected = 'cico_add_funds_bottom_sheet_selected',
   cico_add_funds_bottom_sheet_impression = 'cico_add_funds_bottom_sheet_impression',
+  cico_add_funds_bottom_sheet_ramp = 'cico_add_funds_bottom_sheet_ramp',
```

Comment:
Can we name this analytics event more explicitly? Maybe `cico_add_funds_bottom_sheet_ramp_selected`. The intention is to distinguish between an impression event and an interaction event so when DS run analysis, it's easier for them to guess what's the event is about

Reply:
Yes, I can change that event. good idea